src/libs/actions/Report.js

```diff
@@ -797,8 +797,17 @@ function subscribeToReportCommentPushNotifications() {
 
     // Open correct report when push notification is clicked
     PushNotification.onSelected(PushNotification.TYPE.REPORT_COMMENT, ({reportID}) => {
-        Navigation.setDidTapNotification();
-        Linking.openURL(`${CONST.DEEPLINK_BASE_URL}${ROUTES.getReportRoute(reportID)}`);
+        if (Navigation.isReady()) {
+            // If a chat is visible other than the one we are trying to navigate to, then we need to navigate back
+            if (Navigation.getActiveRoute().slice(1, 2) === ROUTES.REPORT && !Navigation.isActiveRoute(`r/${reportID}`)) {
```

Comment:
Can we use `isActiveRoute()` instead here?

Reply:
I wasn't able to figure out a way since `isActiveRoute` will require a full path (e.g. `r/123`), and all we want to know is if we are on any report.

Reply:
Oh ok I understand now. I wasn't reading this correctly.